src/components/pagination/bl-pagination.test.ts

```diff
@@ -39,8 +39,6 @@ describe("bl-pagination", () => {
     expect(el.itemsPerPage).to.equal(10);
     expect(el.hasJumper).to.equal(false);
     expect(el.hasSelect).to.equal(false);
-    expect(el.jumperLabel).to.equal("Go To");
-    expect(el.selectLabel).to.equal("Show");
```

Comment:
Imho instead of removing the tests, we should verify defaults with localized values